### PR TITLE
Add Advantech WebAccess LoginScanner module

### PR DESCRIPTION
**Pull request by wchen-r7**
## Description

This module allows you to auth to AdvanTech WebAccess.

## Verification

1. Install Advantech WebAccess: http://advcloudfiles.advantech.com/web/Download/webaccess/8.1/AdvantechWebAccessUSANode8.1_20151230.exe
2. Add some users to WebAccess
3. Start msfconsole
4. ```use auxiliary/scanner/http/advantech_webaccess_login```
5. ```set RHOSTS [IP]```
6. Provide some credentials for testing
7. ```run```
8. If a valid user/pass is found, it should print green and report to database

## Demo

webaccess_login_demo

### CODE REVIEW

File: documentation/modules/auxiliary/scanner/http/advantech_webaccess_login.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+## Description
+
+This module allows you to authenticate Advantech WebAccess.
```
Comment (h00die) [inline documentation/modules/auxiliary/scanner/http/advantech_webaccess_login.md]: authenticate **to** Advantech

File: documentation/modules/auxiliary/scanner/http/advantech_webaccess_login.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+**8.0 Download**
+
+http://advcloudfiles.advantech.com/web/Download/webaccess/8.0/AdvantechWebAccessUSANode8.0_20141103_3.4.3.exe
+
+
+## Verification Steps
+
+1. Make sure Advantech WebAccess is up and running
+2. Start ```msfconsole```
+3. ```use auxiliary/scanner/http/advantech_webaccess_login
```
Comment (h00die) [inline documentation/modules/auxiliary/scanner/http/advantech_webaccess_login.md]: forgot ending ```

File: documentation/modules/auxiliary/scanner/http/advantech_webaccess_login.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+4. ```set RHOSTS [IP]```
+5. Set credentials
+6. ```run```
+7. You should see that the module is attempting to log in.
+
```
Comment (h00die) [inline documentation/modules/auxiliary/scanner/http/advantech_webaccess_login.md]: I always like having an example run in the docs as well.  Can you add one, with the full run of choosing module, make sure to try default, and have another say 2 bad logins there

File: documentation/modules/auxiliary/scanner/http/advantech_webaccess_login.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+**8.1 Download**
+
+http://advcloudfiles.advantech.com/web/Download/webaccess/8.1/AdvantechWebAccessUSANode8.1_20151230.exe
+
+**8.0 Download**
+
+http://advcloudfiles.advantech.com/web/Download/webaccess/8.0/AdvantechWebAccessUSANode8.0_20141103_3.4.3.exe
+
+
```
Comment (h00die) [inline documentation/modules/auxiliary/scanner/http/advantech_webaccess_login.md]: Can you either add a comment here, or another heading for Variables, or Defaults where it mentions admin:'' is the default creds?  Just wanna get that captured in the docs as well (even though its in the module description)

File: documentation/modules/auxiliary/scanner/http/advantech_webaccess_login.md (hunk 2 above)
Comment (wchen-r7) [inline documentation/modules/auxiliary/scanner/http/advantech_webaccess_login.md]: Fixed. Thanks!

File: documentation/modules/auxiliary/scanner/http/advantech_webaccess_login.md (hunk 4 above)
Comment (wchen-r7) [inline documentation/modules/auxiliary/scanner/http/advantech_webaccess_login.md]: Fixed. Thanks!

File: documentation/modules/auxiliary/scanner/http/advantech_webaccess_login.md (hunk 1 above)
Comment (wchen-r7) [inline documentation/modules/auxiliary/scanner/http/advantech_webaccess_login.md]: Fixed. Thanks!

File: documentation/modules/auxiliary/scanner/http/advantech_webaccess_login.md (hunk 3 above)
Comment (wchen-r7) [inline documentation/modules/auxiliary/scanner/http/advantech_webaccess_login.md]: A demo is attached. Thanks!

**Comment by h00die** [comment]
rebooting my ESX server, soon as its up i'll knock out #7874 and #7876 together since I already have the software installed and ready to rock!

**Comment by wchen-r7** [comment]
@h00die Thanks! Please take your time :-)

**Comment by h00die** [comment]
```
msf auxiliary(advantech_webaccess_login) > run

[*] Default credential admin:[empty] added to the credential queue for testing.
[+] ADVANTECH_WEBACCESS - Success: 'admin:'
[!] No active DB -- Credential data will not be saved!
[-] ADVANTECH_WEBACCESS - Failed: 'bad:'
[-] ADVANTECH_WEBACCESS - Failed: 'alsobad:'
[*] Scanned 1 of 1 hosts (100% complete)
[*] Auxiliary module execution completed

```

**Comment by h00die** [comment]
# Release Notes

This module adds a login scanner for Advantech WebAccess

**Comment by h00die** [comment]
@wchen-r7 thank for the submission, your code looked very pretty

**Comment by wvu-r7** [comment]
@wchen-r7 does write pretty code. :)

**Pull request merged**